config/config.go

```diff
@@ -48,8 +48,6 @@ type BlockManagerConfig struct {
 	// BatchSubmitMaxTime defines how long should block manager wait for before submitting batch
 	BatchSubmitMaxTime time.Duration `mapstructure:"batch_submit_max_time"`
 	NamespaceID        string        `mapstructure:"namespace_id"`
-	// The size of the batch in blocks. Every batch we'll write to the DA and the settlement layer.
-	BlockBatchSize uint64 `mapstructure:"block_batch_size"`
 	// The size of the batch in Bytes. Every batch we'll write to the DA and the settlement layer.
 	BlockBatchMaxSizeBytes uint64 `mapstructure:"block_batch_max_size_bytes"`
```

Comment:
depends on block size. currently max block size = max batch size. so at least 1 block per batch. empty blocks is around 10kb IIRC, so around 50 per batch in case all blocks are empty.

Reply:
seems that empty block is < 1KB

Reply:
thx